File: 1960-check-if-the-sentence-is-pangram/check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String s) {
        if (s.length() < 26)
            return false;

        for (int i = 0; i < 26; i++) {
            char ch = (char)('a' + i);
            if (!s.contains(String.valueOf(ch)))
                return false;
        }
        return true;
    }
}

File: 1960-check-if-the-sentence-is-pangram/check-if-the-sentence-is-pangram.cpp
class Solution {
public:
    bool checkIfPangram(string s) {
       int f[26]={0};
       for(int i=0;i<s.size();i++)
       {
            f[s[i]-'a']++;
       }
       for(int i=0;i<26;i++)
       {
        if(f[i]==0)
        return false;
       }
       return true;
    }
};